src/components/Keyboard/Alphabet/Alphabet.js

```diff
@@ -1,14 +1,13 @@
 import React from 'react';
 import { connect } from 'react-redux';
-import { bindActionCreators } from 'redux';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import { setGameState } from '../../../actions/gameState';
 import styles from './Alphabet.css';
-import Letter from '../Letter';
+import { Letter } from '../Letter';
 import { LETTER_STATUS_CORRECT, LETTER_STATUS_INCORRECT } from '../../../../consts';
 
-class Alphabet extends React.Component {
+class AlphabetPrimary extends React.Component {
```

Comment:
Why name change? You should restore the previous one, because it makes no sense to change it

Reply:
This is caused Alphabet is connected with redux and I'm using named exports in project.